resources/AddMatch.html

```diff
@@ -124,37 +135,70 @@ <h3 id="addmatch-title">Add a match</h3>
+      <div class="scored-balls-label">Penalties</div>
+      <div class="scored-balls-inputs">
+        <div class="port-label">
+          Fouls:
+        </div>
+        <div class="single-port-input">
+          <button onclick="addTo('fouls', -1);" class="decrement-button" type="button">⬇️</button>
+          <input type="text" id="fouls" name="fouls" pattern="[0-9]*" placeholder="Fouls" class="outer-goal" value="0"
+                 min="0" required>
+          <button onclick="addTo('fouls', 1);" class="increment-button" type="button">⬆️</button>
+        </div>
+        <div class="port-label">
+          Tech Fouls:
+        </div>
+        <div class="single-port-input">
+          <button onclick="addTo('techfouls', -1);" class="decrement-button" type="button">⬇️</button>
+          <input type="text" id="techfouls" name="tech-fouls" pattern="[0-9]*" placeholder="Tech Fouls"
+                 class="outer-goal" value="0" min="0" required>
+          <button onclick="addTo('techfouls', 1);" class="increment-button" type="button">⬆️</button>
+          <div class="clearfix"></div>
+        </div>
+      </div>
+    </div>
+
     <div class="level-shield-div">
-      <label for="estopped">Was the robot emergency stopped?</label> <input type="checkbox" id="estopped" name="estopped" value="true">
+      <label for="estopped">Was the robot emergency stopped/disabled/turned off (RSL solid or off)?</label> <input
+            type="checkbox" id="estopped" name="estopped" value="true">
     </div>
 
     <div class="level-shield-div">
-      <label for="yellowcard">Did the robot get a yellow card?</label> <input type="checkbox" id="yellowcard" name="yellowcard" value="true">
+      <label for="yellowcard">Did the robot get a yellow card?</label> <input type="checkbox" id="yellowcard"
+                                                                              name="yellowcard" value="true">
     </div>
 
     <div class="level-shield-div">
-      <label for="redcard">Did the robot get a red card?</label> <input type="checkbox" id="redcard" name="redcard" value="true">
+      <label for="redcard">Did the robot get a red card?</label> <input type="checkbox" id="redcard" name="redcard"
+                                                                        value="true">
     </div>
 
 
     <div class="level-shield-div">
-      <label for="borked">Was the robot disabled?</label> <input type="checkbox" id="borked" name="borked" value="true">
+      <label for="borked">Did the robot attempt to climb and fail (fall off/invalidated)?</label> <input type="checkbox"
+                                                                                                         id="borked"
+                                                                                                         name="borked"
+                                                                                                         value="true">
```

Comment:
Shouldn't this set the `monkeybarstate` to `none`? That way, we can remove the "failed" option from the climbing menu and just override it with this. Or did you want the climbing menu to be set to the level that was attempted?

Reply:
I want it to be separate so we can see if robots say attempt high/traversal rungs but fall off and then get back up onto the mid rung or something can be marked as both failed climb + mid rung. It was a bit more common than I'd previously anticipated at MBR.

Reply:
Alright, that's fine, but let's remove the "failed" option from the climbing menu to avoid confusion.